### PR TITLE
FMT: Sanitize input filenames to make cl-safe

### PR DESCRIPTION
**Pull request by joshicola**
* FIX: Change to Dockerfile to reference unexpired trusty dependency
* FIX: Sanitize input filenames to make them command-line safe.
* Note: The referenced function `create_sanitized_filepath` is in [`hcp-base`](https://github.com/flywheel-apps/hcp-base/blob/master/utils/gear_preliminaries.py#L320) and shared across the HCP-SUITE.

### CODE REVIEW

**Comment by joshicola** [comment]
@naterichman , the `trusty` dependency in the Dockerfile, `http://neurodebian.ovgu.de/lists/trusty.us-ca.full` will not be necessary in future versions of the HCP-SUITE.  Unfortunately, some component currently depends on it for now.

File: utils/args/GenericfMRIVolumeProcessingPipeline.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from collections import OrderedDict
 
 from tr import tr
+from utils.gear_preliminaries import create_sanitized_filepath
```
Comment (naterichman) [inline utils/args/GenericfMRIVolumeProcessingPipeline.py:14]: I don't see where this module is.

**Review by naterichman — COMMENTED** [review]
The create_sanitized_filepath seems like it was the biggest change here, but I can't seem to find it.  I would just like to see what it's doing otherwise, it looks good to me.

File: utils/args/GenericfMRIVolumeProcessingPipeline.py (hunk 1 above)
Comment (joshicola) [reply utils/args/GenericfMRIVolumeProcessingPipeline.py:14]: Thanks, @naterichman ,

The reference to the `create_sanitized_filepath` is listed in the "NOTE:" above, and again [here](https://github.com/flywheel-apps/hcp-base/blob/master/utils/gear_preliminaries.py#L320).

Comment (naterichman) [reply utils/args/GenericfMRIVolumeProcessingPipeline.py:14]: Totally missed that note!

**Review by naterichman — APPROVED** [review]
Sorry about missing that note!

**Pull request merged**